integration/guessinggame/guessing_contract_bytecode.go

```diff
@@ -0,0 +1,3 @@
050565b60008135905061084981610dc6565b92915050565b60006020828403121561086557610864610ccc565b5b600061087384828501610810565b91505092915050565b60006020828403121561089257610891610ccc565b5b60006108a084828501610825565b91505092915050565b6000602082840312156108bf576108be610ccc565b5b60006108cd8482850161083a565b91505092915050565b6108df81610b5d565b82525050565b6108ee81610bc0565b82525050565b6108fd81610bd2565b82525050565b6000610910602283610b4c565b915061091b82610cd1565b604082019050919050565b6000610933601a83610b4c565b915061093e82610d20565b602082019050919050565b6000610956602983610b4c565b915061096182610d49565b604082019050919050565b61097581610b7b565b82525050565b61098481610ba9565b82525050565b61099b61099682610ba9565b610c33565b82525050565b6109aa81610bb3565b82525050565b60006109bc828561098a565b6020820191506109cc828461098a565b6020820191508190509392505050565b60006020820190506109f160008301846108d6565b92915050565b6000604082019050610a0c60008301856108d6565b610a1960208301846108d6565b9392505050565b6000606082019050610a3560008301866108d6565b610a4260208301856108d6565b610a4f60408301846108f4565b949350505050565b6000604082019050610a6c60008301856108d6565b610a79602083018461097b565b9392505050565b6000602082019050610a9560008301846108e5565b92915050565b60006020820190508181036000830152610ab481610903565b9050919050565b60006020820190508181036000830152610ad481610926565b9050919050565b60006020820190508181036000830152610af481610949565b9050919050565b6000602082019050610b10600083018461096c565b92915050565b6000602082019050610b2b600083018461097b565b92915050565b6000602082019050610b4660008301846109a1565b92915050565b600082825260208201905092915050565b6000610b6882610b89565b9050919050565b60008115159050919050565b600061ffff82169050919050565b600073ffffffffffffffffffffffffffffffffffffffff82169050919050565b6000819050919050565b600060ff82169050919050565b6000610bcb82610be4565b9050919050565b6000610bdd82610ba9565b9050919050565b6000610bef82610bf6565b9050919050565b6000610c0182610b89565b9050919050565b6000610c1382610b7b565b915061ffff821415610c2857610c27610c6e565b5b600182019050919050565b6000819050919050565b6000610c4882610ba9565b9150610c5383610ba9565b925082610c6357610c62610c9d565b5b828206905092915050565b7f4e487b7100000000000000000000000000000000000000000000000000000000600052601160045260246000fd5b7f4e487b7100000000000000000000000000000000000000000000000000000000600052601260045260246000fd5b600080fd5b7f4f6e6c79206f776e65722063616e2063616c6c20746869732066756e6374696f60008201527f6e2e000000000000000000000000000000000000000000000000000000000000602082015250565b7f436865636b2074686520746f6b656e20616c6c6f77616e63652e000000000000600082015250565b7f42616c616e6365206d757374206265207a65726f20746f207365742061206e6560008201527f77207461726765742e0000000000000000000000000000000000000000000000602082015250565b610da181610b6f565b8114610dac57600080fd5b50565b610db881610ba9565b8114610dc357600080fd5b50565b610dcf81610bb3565b8114610dda57600080fd5b5056fea264697066735822122005c22e01637640456bd0c41fd50670919de1ac74a64a0964cfa45a32615c83bc64736f6c63430008070033`
```

Comment:
ContractByteCode would need to change if we do add in a call to setNewTarget in the constructor ofc.

Reply:
yes, of course.
have a look now, pls